pkg/volume/volume.go

```diff
@@ -86,8 +86,8 @@ type Builder interface {
 	GetAttributes() Attributes
```

Comment:
In description of `Attributes struct` few lines above, there is left `builder` instead of mounter: 

```
 // Attributes represents the attributes of this builder.
```

(github does not allow me to comment lines that are not part of a commit)